Algorithm-Return-Largest-Numbers-in-Arrays.md

```diff
@@ -70,29 +88,34 @@ function largestOfFour(arr) {
 `Math.max([9, 43, 20, 6]); // Resulting in 43`<br>Alas, it doesn't.
 
 - To do the work of accepting arguments in the shape of an array, there is this `Function.apply` method, but it complicates things a bit by *invoking* the *context* function.<br>
-i.e. `Math.max.apply(null, [9, 43, 20, 6]);` would invoke something like a `Max.max` method. What we're looking for... almost. 
+i.e. `Math.max.apply(null, [9, 43, 20, 6]);` would invoke something like a `Max.max` method. What we're looking for... almost.
 
-> Here we're passing `null` as the *context* of the `Function.apply` method as `Math.max` doesn't need any context.
+Here we're passing `null` as the *context* of the `Function.apply` method as `Math.max` doesn't need any context.
 
-- Since `arr.map` expects a callback function, not just an expression, we create a function out of the previous expression by using the `Function.bind` method. 
+- Since `arr.map` expects a callback function, not just an expression, we create a function out of the previous expression by using the `Function.bind` method.
 - Since, `Function.apply` is a static *method* of the same `Function` *object*, we can call `Function.prototype.bind` on `Function.apply` i.e. `Function.apply.bind`.

```

Comment:
> NOTE TO SELF AND OTHERS
*I wrote it wrong*. `Function.apply` is **NOT** a static *method* of `Function` *object*. It's a *method* on `Function` `prototype` i.e. `Function.prototype.apply`.